service/frontend/adminHandler.go

```diff
@@ -1048,13 +1047,6 @@ func (adh *adminHandlerImpl) validateGetWorkflowExecutionRawHistoryV2Request(
 		return &types.BadRequestError{Message: "Invalid PageSize."}
 	}
 
-	if request.StartEventID == nil &&
-		request.StartEventVersion == nil &&
-		request.EndEventID == nil &&
-		request.EndEventVersion == nil {
-		return &types.BadRequestError{Message: "Invalid event query range."}
-	}
```

Comment:
@yux0 Any idea why this case is not allowed? 
From code (https://github.com/uber/cadence/blob/0.22.x/service/frontend/adminHandler.go#L1063) looks like it can be handled. I also tried locally by re-replicating a workflow without specifying any of the four fields and it can work.
Please let me know if I missed anything. Thanks!

Reply:
the start event id/version or end event id/version is required to identify a branch. So this is needed

Reply:
Synced offline. This check is not necessary.